File: Copilot/1. Algoritmo Básico Ordenamiento por Burbuja/Python/burbuja.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr

nums = list(map(int, input("Ingresa números separados por espacio: ").split()))
print("Lista ordenada:", bubble_sort(nums))

File: Copilot/1. Algoritmo Básico Ordenamiento por Burbuja/Golang/burbuja.go
package main

import (
    "bufio"
    "fmt"
    "os"
    "strconv"
    "strings"
)

func bubbleSort(arr []int) {
    n := len(arr)
    for i := 0; i < n; i++ {
        for j := 0; j < n-i-1; j++ {
            if arr[j] > arr[j+1] {
                arr[j], arr[j+1] = arr[j+1], arr[j]
            }
        }
    }
}

func main() {
    reader := bufio.NewReader(os.Stdin)
    fmt.Print("Ingresa números separados por espacio: ")
    input, _ := reader.ReadString('\n')
    input = strings.TrimSpace(input)
    parts := strings.Split(input, " ")

    var arr []int
    for _, p := range parts {
        num, _ := strconv.Atoi(p)
        arr = append(arr, num)
    }

    bubbleSort(arr)
    fmt.Println("Lista ordenada:", arr)
}
